### ISSUE TITLE
Change beaconFilterinTermsResults type to enum

### ISSUE THREAD

**Issue by SergiAguilo** [ga4gh-beacon/beacon-v2#77]
In the [beaconFilteringTermsResults.json](https://github.com/ga4gh-beacon/beacon-v2/blob/main/framework/json/responses/sections/beaconFilteringTermsResults.json) the "type" property of the FilteringTerm is ambiguous as it only requires an string.

I suggest to enumerate the three types of filtering terms to constrain the property.

Currently, the property is the following:
```json
"type":{
    "description": "Either \"custom\", \"alphanumeric\" or ontology/terminology full name. TODO: An ontology ... with a registered prefix does not need a full name so one may better use CURIE to indicate that the resource can be retrieved from the id. This also will allow to provide an enum here.",
    "examples": [
        "Human Phenotype Ontology",
        "alphanumeric"
    ],
    "type": "string"
}
```

My suggestion is:

```json
"type":{
    "description": "Either \"custom\", \"alphanumeric\" or \"ontology\" .",
    "enum": [
        "custom",
        "alphanumeric",
        "ontology"
    ]
    "example": "alphanumeric",
    "type": "string"
}
```
The "custom" type already includes any other type of filters. This enumeration will help to standardize the filters endpoint in the different instances of Beacon, specially when gathering them in the Beacon Network.

**Comment by costero-e** [comment]
+1 to this @SergiAguilo

**Comment by mbaudis** [comment]
+1

**Comment by costero-e** [comment]
I addressed this issue by creating this new PR #90 . @SergiAguilo I couldn't add you as reviewer because I couldn't tag you, so I added @redmitry instead
